index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
```

Comment:
before game over we much display the name of the player who wins as the pop up
and the one who loses must get to know about he or she not winning and trying again